openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-scriptrunner/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "@openc3/tool-common": "5.1.2-beta0",
-    "ace-builds": "1.13.2",
+    "ace-builds": "1.13.0",
```

Comment:
Starting with 1.31.1 it doesn't render correctly: https://github.com/ajaxorg/ace/issues/5012